modules/command-engine/javadsl/src/main/scala/surge/javadsl/common/AggregateRefBaseTrait.scala

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2017-2021 UKG Inc. <https://www.ukg.com>
+
+package surge.javadsl.common
+
+import akka.actor.ActorRef
+import io.opentracing.Tracer
+import surge.internal.persistence.{ AggregateRefTrait, PersistentActor }
+
+import java.util.Optional
+import java.util.concurrent.CompletionStage
+import scala.compat.java8.FutureConverters
+import scala.compat.java8.OptionConverters._
+
+import scala.concurrent.ExecutionContext
+
+trait AggregateRefEvent[Agg, Event] {
```

Comment:
I don't think we're actually getting anything from this since we've got an `event.AggregateRef` trait for the event specific handlers already